charts/values.yaml

```diff
@@ -34,6 +34,12 @@ feature:
   tunnelIpv6Subnet: "fd11::/112"
   ## @param feature.tunnelDetectMethod Tunnel base on which interface [`defaultRouteInterface`, `interface=eth0`]
   tunnelDetectMethod: "defaultRouteInterface"
+  ## @param feature.enableGatewayReplyRoute  the gateway node reply route is enabled, which should be enabled for spiderpool
+  enableGatewayReplyRoute: false
+  ## @param feature.gatewayReplyRouteTable  host Reply routing table number on gateway node
+  gatewayReplyRouteTable: 600
+  ## @param feature.gatewayReplyRouteMark  host iptables mark for reply packet
```

Comment:
host iptables mark for reply packet  on gateway node

Reply:
> host iptables mark for reply packet on gateway node

done